### ISSUE TITLE
Missing trailing value in merge_sorted

### ISSUE THREAD

**Issue by mrocklin** [pytoolz/cytoolz#55]
```python
In [1]: data = [[(1, 2), (0, 4), (3, 6)], [(5, 3), (6, 5), (8, 8)], [(9, 1), (9, 8), (9, 9)]]

In [2]: import toolz

In [3]: list(toolz.merge_sorted(*data, key=lambda x: x[1]))
Out[3]: [(9, 1), (1, 2), (5, 3), (0, 4), (6, 5), (3, 6), (8, 8), (9, 8), (9, 9)]

In [4]: import cytoolz

In [5]: list(cytoolz.merge_sorted(*data, key=lambda x: x[1]))
Out[5]: [(9, 1), (1, 2), (5, 3), (0, 4), (6, 5), (3, 6), (8, 8), (9, 8), 9]
```

**Comment by eriknw** [comment]
Fixed via #56.  Thanks for the report, @mrocklin.  Should I do a bugfix release?

**Comment by mrocklin** [comment]
No rush on my end.

**Comment by eriknw** [comment]
Okay.  I just released cytoolz 0.7.2, which implements toolz version 0.7.1.
